README.md

```diff
@@ -11,6 +11,15 @@ a [Sails v1](https://sailsjs.com) application
 + [Community support options](https://sailsjs.com/support)
 + [Professional / enterprise options](https://sailsjs.com/enterprise)
 
+### Quick Start
++ Install [MongoDB](https://www.mongodb.com/try/download/community)
++ Install Sails.js: `npm install -g sails`
++ Install dependencies: `npm install`
++ Run: `sails lift`
```

Comment:
Let's also add: `sails lift` or `sails lift --port NUMBER` in case someone gets this error `EADDRINUSE` (which is what I encountered)

Reply:
yes sir